locale/it_IT/electrum.po

```diff
@@ -38,17 +38,17 @@ msgstr " {} {} pronto per essere inviato."
 #: electrum_dash/gui/qt/protx_qt.py:548
 #, python-format
 msgid "%s %s registered DIP3 Masternodes."
-msgstr ""
+msgstr "%s %s Masternode DIP3 registrati."
 
 #: electrum_dash/gui/qt/protx_qt.py:967
-#, fuzzy, python-format
+#, python-format
 msgid "%s Dip3 Masternode Info"
-msgstr "Masternode.conf"
+msgstr "%s Info Msternode Dip3"
```

Comment:
a typo, must be `Masternode` instead of `Msternode`

Reply:
fixed in b028cf7